doc/api/thalireplicationmanager.md

```diff
@@ -0,0 +1,106 @@
+# The `ThaliReplicationManager` class
```

Comment:
Done as noted:

The `ThaliReplicationManager` class handles database replication between devices, using [PouchDB](http://pouchdb.com/) and the Thali Cordova bridge `ThaliEmitter` class.  This class is meant solely for the purpose of demonstrating [Thali Story 0](http://thaliproject.org/stories) and will be dramatically enhanced in the future.